### ISSUE TITLE
registry proxy failing to start

### ISSUE THREAD

**Issue by debianmaster** [rancher/rio#235]
```sh
NAME                               READY   STATUS             RESTARTS   AGE
autoscaler-696866b8f7-w2f7h        1/1     Running            0          8m2s
build-controller-94b48b58f-tvxmn   2/2     Running            0          7m28s
buildkit-7f46884f98-r5tkk          2/2     Running            0          7m30s
cert-manager-57b4f876c5-qw5x6      1/1     Running            0          7m32s
grafana-68c6d8494d-l4t4w           2/2     Running            0          7m52s
istio-citadel-d76685f74-m77pg      1/1     Running            0          7m49s
istio-gateway-fvbj7                2/2     Running            0          7m48s
istio-gateway-h9r8n                2/2     Running            0          7m48s
istio-gateway-kwxpd                2/2     Running            0          7m48s
istio-gateway-nxd4j                2/2     Running            0          7m48s
istio-gateway-p6zhf                2/2     Running            0          7m48s
istio-pilot-85b894cd75-tdbjq       2/2     Running            0          7m46s
istio-telemetry-6b686cbbf5-fc8nn   2/2     Running            0          7m55s
kiali-7d4b6dc78c-x954s             2/2     Running            0          7m35s
prometheus-59c5655d9c-6gc2b        1/1     Running            0          8m2s
registry-6c4fddb8b6-vmxj2          2/2     Running            0          7m19s
registry-proxy-dspdq               0/1     CrashLoopBackOff   0          7m59s
registry-proxy-g9gtl               0/1     CrashLoopBackOff   0          7m59s
registry-proxy-hl25f               0/1     Error              4          85s
registry-proxy-htpq2               0/1     CrashLoopBackOff   0          7m59s
registry-proxy-t5kwq               0/1     CrashLoopBackOff   0          7m59s
rio-controller-876548768-ljs84     1/1     Running            0          8m17s
webhook-5f6fc8f6f9-jwx72           2/2     Running            0          7m18s
chakwifi$ k logs -f registry-proxy-dspdq -n rio-system
+ trap exit TERM INT
+ cat /proc/sys/net/ipv4/ip_forward
+ '[' 0 '!=' 1 ]
+ exit 1
```

**Comment by JasonvanBrackel** [comment]
Can confirm this.  klipper container is stuck

```
jason@vblinux:~$ kubectl logs registry-proxy-tt9lz
+ trap exit TERM INT
+ cat /proc/sys/net/ipv4/ip_forward
+ '[' 0 '!=' 1 ]
+ exit 1
jason@vblinux:~$ kubectl logs registry-proxy-2pffm
+ trap exit TERM INT
+ cat /proc/sys/net/ipv4/ip_forward
+ '[' 0 '!=' 1 ]
+ exit 1
```

**Comment by virtuman** [comment]
same for me, running on CENTOS 7.8 as well as rancheros 1.5.1. Message seems to contradict what we've all been taught that ip_forward needs to be enabled in the dockerized environment.

```
$ kubectl -n rio-system logs -f --previous pod/registry-proxy-s8hnz
+ trap exit TERM INT
+ cat /proc/sys/net/ipv4/ip_forward
+ '[' 0 '!=' 1 ]
+ exit 1
```

when examining the server host, it appears that ip_forward is correctly set to 1 on all of our servers:
```
[root@dev-k8s-node3 ~]# cat /proc/sys/net/ipv4/ip_forward
1
```

**Comment by JasonvanBrackel** [comment]
This worked fine in a katacoda playground.  (kubeadm based cluster).

This did not work in my RKE cluster created with Rancher 2.1.x that has been upgraded several times over to 2.2.3 and is using --acme-domain certs

**Comment by resolute21** [comment]
I have the same error on registry-proxy, tried docker run --sysctl net.ipv4.ip_forward=1 and it still doesn't work.

My cluster is a custom one with CentOS 7.

**Comment by StrongMonkey** [comment]
v0.1.1-rc2 should fix the issue.

**Comment by resolute21** [comment]
Solved on v0.1.1-rc2 as indicated.

NAME                               READY   STATUS    RESTARTS   AGE
autoscaler-696866b8f7-2jt8j        1/1     Running   0          5m14s
build-controller-94b48b58f-mn9nw   2/2     Running   0          4m45s
buildkit-7f46884f98-w7qrl          2/2     Running   0          5m7s
cert-manager-57b4f876c5-lb9x5      1/1     Running   0          4m44s
grafana-68c6d8494d-dhf5g           2/2     Running   0          5m4s
istio-citadel-d76685f74-wnwjb      1/1     Running   0          5m1s
istio-gateway-wwns8                2/2     Running   0          4m57s
istio-pilot-85b894cd75-wscnc       2/2     Running   0          4m51s
istio-telemetry-6b686cbbf5-fjpqp   2/2     Running   0          5m4s
kiali-7d4b6dc78c-x2wt2             2/2     Running   0          4m44s
prometheus-59c5655d9c-gpv9p        1/1     Running   0          5m14s
registry-6c4fddb8b6-tbr2c          2/2     Running   0          4m37s
registry-proxy-hhbgm               1/1     Running   0          5m7s
rio-controller-7c8668f55-cgjbc     1/1     Running   0          5m39s
webhook-5f6fc8f6f9-nfdcb           2/2     Running   0          4m37s

**Comment by debianmaster** [comment]
Fixee
